QualificationChecker/Controllers/CandidatesController.cs

```diff
@@ -11,100 +13,63 @@ public partial class CandidatesController : ControllerBase
         [HttpPost("showQualified/")]
         public IActionResult ShowQualifiedCandidates([FromBody] MatchRequest request)
```

Comment:
Note: controller methods should be extremely short, most of the work wrapped up elsewhere

Basic steps in controller
  - validation
  - action (if any, like save to DB)
  - return model